testing/examples/author_tests.rs

```diff
@@ -15,29 +15,36 @@
 
 //! Tests for the author rpc interface functions.
 
-use kitchensink_runtime::{AccountId, Runtime, Signature};
+use kitchensink_runtime::AccountId;
 use sp_core::sr25519::Pair;
 use sp_keyring::AccountKeyring;
+use sp_runtime::traits::GetRuntimeBlockType;
 use std::{thread, time::Duration};
 use substrate_api_client::{
 	ac_node_api::EventDetails,
-	ac_primitives::{
-		AssetTipExtrinsicParams, ExtrinsicSigner as GenericExtrinsicSigner, SignExtrinsic,
-	},
+	ac_primitives::{ExtrinsicSigner as GenericExtrinsicSigner, SignExtrinsic, SubstrateConfig},
 	extrinsic::BalancesExtrinsics,
 	rpc::{HandleSubscription, JsonrpseeClient},
 	Api, SubmitAndWatch, SubmitAndWatchUntilSuccess, SubmitExtrinsic, TransactionStatus, XtStatus,
 };
 
-type ExtrinsicSigner = GenericExtrinsicSigner<Pair, Signature, Runtime>;
+// This example run against a specific node.
```

Comment:
You don't need to add these comments to the testing files. They are meant for CI only, not for users. Rather, the comments may get outdated and rather confuse the developer.